parser/src/command/glacier.rs

```diff
@@ -45,7 +45,7 @@ impl GlacierCommand {
                 Some(Token::Quote(s)) => {
                     let source = s.to_owned();
                     if source.starts_with("https://gist.github.com/") {
-                        return Ok(Some(GlacierCommand { source }));
+                        Ok(Some(GlacierCommand { source }))
                     } else {
                         return Err(toks.error(ParseError::InvalidLink));
```

Comment:
Why is this `return` not removed?

Reply:
Because clippy didn't say to remove it, but I suppose it may as well be removed. Will do